src/Raven.Studio/languageService/test/grammar/groupByStatement.spec.ts

```diff
@@ -0,0 +1,49 @@
+﻿import {parseRql} from "../../src/parser";
+import {
+    GroupByStatementContext,
+    ParameterWithOptionalAliasContext,
+} from "../../src/generated/RqlParser";
+
+describe("GROUP BY statement parser", function () {
```

Comment:
remove space after `function` - in all places

Reply:
done